packages/config/config/devices/0x027a/zse29.json

```diff
@@ -16,6 +16,96 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "PIR Control",
+			"maxNodes": 4
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Value",
+			"description": "Set the value of Basic Set commands sent to the devices associated in Group 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Enable / Disable Motion Reports",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "Motion Sensitivity",
+			"description": "Adjust motion sensitivity where 10 is the most sensitive setting.",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 10,
```

Comment:
The manual doesn't specify a default value. Where did you get 10? Generally, setting the maximum sensitivity is a bad idea for PIR devices. Usually they default to the medium range.

Reply:
Got this from the OZW config file, but I don't see this mentioned in the reference link you sent from Zooz (thanks for that).  I'm not sure what this is from personal experience, so I'll remove the default value.

Reply:
Sorry, I see I can't remove the default value.  I'm keeping it at 10 (unless you recommend an arbitrary value), since the only independent info I have is the OZW config file.

Reply:
BTW @AlCalzone I think this points to an undesired aspect of writing these config files.  For read-only parameters, defaultValue must still be specified.  But I don't think defaultValue makes sense in this context.  I had to pick arbitrary defaultValue settings for the read-only parameters @blhoward2 pointed out (otherwise, I failed the lint check); but these values just return physical settings on the device.

Reply:
Good point. I'll remove this necessity.